### ISSUE TITLE
[FP] get SchemaType option

### ISSUE THREAD

**Issue by rafaelcorreiapoli** [szokodiakos/typegoose#375]
## Which feature

```js
var numberSchema = new Schema({
 userId: {
    type: ObjectId,
    get: v => v.toString(),
  }
});
```

https://mongoosejs.com/docs/schematypes.html#schematype-options

## Why do you want it

I need to convert all my ObjectIds to strings

## Additional Notes

I need to preserve the name of the field (in this case, `userId`), so class getters and setters would not work in this case

Thanks!

**Comment by hasezoey** [comment]
already done that:
https://github.com/szokodiakos/typegoose/pull/260
https://github.com/hasezoey/typegoose/issues/5
-> it will be included when 6.0.0 releases

**Comment by rafaelcorreiapoli** [comment]
Nice! thanks :) I will close the issue
